Description: Set 'no owner password' by setting the owner pw to the user pw.
 This patch solves the issue of generating random owner passwords by the
 setEncryption routine of itext if the supplied owner password has zero length.

 According to the PDF specification, the meaning of 'no owner password' is archived
 by setting the owner password equal to the user password.
Author: Johann Felix Soden <johfel@gmx.de>
Bug-Debian: http://bugs.debian.org/607018

--- a/pdftk/pdftk.cc
+++ b/pdftk/pdftk.cc
@@ -2193,6 +2193,10 @@
 			prompt_for_password( "user", "the output PDF", m_output_user_pw );
 		}
 
+		if( m_output_owner_pw.empty() && !m_output_user_pw.empty() ) {
+			m_output_owner_pw= m_output_user_pw;
+		}
+
 		jbyteArray output_owner_pw_p= JvNewByteArray( m_output_owner_pw.size() ); {
 			jbyte* pw_p= elements(output_owner_pw_p);
 			memcpy( pw_p, m_output_owner_pw.c_str(), m_output_owner_pw.size() ); 
